Gemfile

```diff
@@ -2,7 +2,7 @@ ruby '>=2.3.1'
 source 'https://rubygems.org'
 
 # Middleman
-gem 'activesupport', '5.2.4.3'
+gem 'activesupport', '5.0.7.2'
```

Comment:
Worth noting I had similar build errors to @jaredc, I'm ok with this downgrade. @evanhsu I also noticed that when the build failed the `macbuild.sh` script opened up my old `./build` directory so it wasn't _immediately obvious_ that something went wrong, until I inspected my console about the super fast build.

Reply:
Yeah I was able to reproduce the failed build by cleaning up my existing docker container for this project and running a clean build with `activesupport:5.2.4.3`.

Without running a clean build, my pre-existing container was used and the build worked just fine.